src/modules/integrator-guide/pages/data-privacy/data-privacy-console.adoc

```diff
@@ -2,78 +2,74 @@ include::ROOT:partial$variables.adoc[]
 
 = Data Privacy Console
 
-The data privacy console enables you to have access to your personal data by which you can request the company to provide you with access to your personal data allowing you to request modifications, deletions, and exports.
-The requests concerning the treatment of your personal data (deletion, correction, and export) are then escalated to the *data privacy officer (DPO)*.
+The data privacy console enables you to access your personal data.
+It provides a platform where you can make a modification, deletions, and export requests to the company concerning your data.
+These requests(deletion, correction, and export) concerning the treatment of your data are then escalated to the xref:data-privacy/data-privacy-officer.adoc[data privacy officer (DPO)].
 
-This feature is also enabled to allow all users to browse locations of where the data is stored.
+This feature is also enabled to allow all users to browse locations where the data is stored.
 
 .The screenshot shows the overview of the data privacy console.
 image::data-privacy/data-privacy-console.png[role="data-zoomable"]
 
-
-
-
-
-
-
-
-
-
-
-
-
 == Features of the Privacy Console
 
 * Data Overview
 * Privacy Requests
-* Data Export
-* Data Deletion
 
 == Data Overview
 
 Data overview allows you to review your profile information and your personal data stored in Corteza applications.
 
 image::data-privacy/data-overview.png[role="data-zoomable"]
 
-
-* Profile Information - Contact information,names, surnames, nickname, profile picture.
+* Profile Information - Contact information, names, surnames, nickname, profile picture.
 
 * Data Application - Personal data stored in your {PRODUCT_NAME} instance.
```

Comment:
Should we show what this interface looks like? Perhaps we could add some screenshots for both the profile info and data application

Reply:
okay